src/vs/workbench/contrib/debug/browser/debug.contribution.ts

```diff
@@ -618,9 +619,15 @@ configurationRegistry.registerConfiguration({
 			description: nls.localize('debug.disassemblyView.showSourceCode', "Show Source Code in Disassembly View.")
 		},
 		'debug.autoExpandLazyVariables': {
-			type: 'boolean',
-			default: false,
-			description: nls.localize('debug.autoExpandLazyVariables', "Automatically show values for variables that are lazily resolved by the debugger, such as getters.")
+			type: 'string',
+			enum: ['auto', 'on', 'off'],
+			default: 'auto',
+			enumDescriptions: [
+				nls.localize('debug.autoExpandLazyVariables.auto', "When in screen reader optimized mode, automatically expand lazy variables."),
+				nls.localize('debug.autoExpandLazyVariables.on', "Always automatically expand lazy variables."),
+				nls.localize('debug.autoExpandLazyVariables.off', "Never automatically expand lazy variables.")
+			],
+			description: nls.localize('debug.autoExpandLazyVariables', "Controls if variables, such as getters, are automatically resolved and expanded by the debugger.")
```

Comment:
I think the "variables that are lazily resolved" was an important part of this description

Reply:
My suggestion: https://github.com/microsoft/vscode/pull/221752